examples/databases/src/sqlx.rs

```diff
@@ -1,13 +1,18 @@
-use rocket::{Rocket, State, Build, futures};
+use rocket::{Rocket, Build, futures};
 use rocket::fairing::{self, AdHoc};
 use rocket::response::status::Created;
 use rocket::serde::{Serialize, Deserialize, json::Json};
 
+use rocket_db_pools::{sqlx, Database};
+
 use futures::stream::TryStreamExt;
 use futures::future::TryFutureExt;
-use sqlx::ConnectOptions;
 
-type Db = sqlx::SqlitePool;
+#[derive(rocket_db_pools::Database)]
+#[database(name = "sqlx")]
+struct Db(sqlx::SqlitePool);
+
+type Connection = rocket_db_pools::Connection<Db>;
```

Comment:
The syntax with an associated type doesn't work for this, since it must be fully qualified: <https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=dc2bdcdce2130b927224fec4f20115aa>.

Reply:
Not of the trait, but of the type directly.

Reply:
Unfortunately, inherent associated types [are unstable](https://github.com/rust-lang/rust/issues/8995) :(